osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -194,6 +187,9 @@ protected override bool OnMouseMove(MouseMoveEvent e)
 
         protected override bool OnKeyDown(KeyDownEvent e)
         {
+            if (OverlayActivationMode.Value == OverlayActivation.Disabled)
+                return false;
```

Comment:
This feels very weird. So now the volume overlay can still be displayed by using the keybinds despite activation mode being "disabled"? Why not do it in a similar manner to `Toolbar`, which is overriding `UpdateState` and blocking `State.Value = Visible` updates by reverting back to `Hidden`.

That will require `VolumeOverlay.Adjust` to increase/decrease volume regardless of visibility state.

Reply:
Also to expand a bit more on this, generally displaying volume overlay would obscure part of the screen mid-game, so it's better to have it take effect but not display at all.